coalib/output/ConsoleInteraction.py

```diff
@@ -250,7 +252,9 @@ def print_result(console_printer,
     console_printer.print(format_lines(result.message))
 
     if interactive:
-        cli_actions = CLI_ACTIONS
+        # FIXME: Handle duplicate actions of same type across
+        # NATIVE_ACTIONS and result.actions
+        cli_actions = NATIVE_ACTIONS + tuple(result.actions)
```

Comment:
> Ah, do you mean the native actions are also the parts of the result.actions field implicitly and the cli_actions just become equivalent to result.actions?

Ah no sorry I just meant we have to do inside `Result.__init__`
```
self.results = []
```
that's all ;)